contrib/babelfishpg_tsql/sql/ownership.sql

```diff
@@ -365,7 +371,8 @@ CAST(CASE WHEN Ext.type = 'R' THEN NULL ELSE Ext.credential_id END AS INT) AS cr
 CAST(CASE WHEN Ext.type = 'R' THEN 1 ELSE Ext.owning_principal_id END AS INT) AS owning_principal_id,
 CAST(CASE WHEN Ext.type = 'R' THEN 1 ELSE Ext.is_fixed_role END AS sys.BIT) AS is_fixed_role
 FROM pg_catalog.pg_roles AS Base INNER JOIN sys.babelfish_authid_login_ext AS Ext ON Base.rolname = Ext.rolname
-WHERE (pg_has_role(suser_id(), 'sysadmin'::TEXT, 'MEMBER')
+WHERE (bbf_is_member_of_role_nosuper(suser_id(), 'sysadmin') 
+  OR bbf_is_member_of_role_nosuper(suser_id(), 'securityadmin')
```

Comment:
Should we use pg_has_role? Because anyway superuser will have all privileges/memberships.

Reply:
+1

Reply:
Right, but isn't it better to give the permission to the required role only?

Reply:
I think we will get the same behaviour with `pg_has_role`.

Reply:
will get same behaviour but the check will pass since it's superuser but here we don't want to check this, we want to check if login is member of sysadmin/securityadmin or not which will get not actually get tested since it will satisfy the superuser condition first.